boilerplate/app/components/Toggle/Switch.tsx

```diff
@@ -0,0 +1,235 @@
+import React from "react"
+import { Image, ImageStyle, Platform, StyleProp, TextStyle, View, ViewStyle } from "react-native"
+import Animated, {
+  Extrapolation,
+  interpolate,
+  useAnimatedStyle,
+  withTiming,
+} from "react-native-reanimated"
+import { colors } from "../../theme"
+import { iconRegistry } from "../Icon"
+import { isRTL } from "app/i18n"
+import { $inputOuterBase, BaseToggleInputProps, Toggle, ToggleProps } from "./Toggle"
+
+export interface SwitchToggleProps extends Omit<ToggleProps<SwitchInputProps>, "ToggleInput"> {
+  /**
+   * Switch-only prop that adds a text/icon label for on/off states.
+   */
+  switchAccessibilityMode?: "text" | "icon"
```

Comment:
Should we change this to just be `accessibilityMode` since it belongs to the `Switch` component?

Reply:
agreed!